src/index.html

```diff
@@ -3260,6 +3269,56 @@ <h4>Indicator chart title</h4>
             <div class="tooltip__notch"></div>
           </div>
         </div>
+        <div class="styles__area_block test">
+          <div class="map-tooltip">
+            <div class="map-tooltip__geography-chip bg-primary">
+              <div>Loading..</div>
+            </div>
+            <div class="map-tooltip__name">
+              <div>Loading...</div>
+            </div>
+            <div class="map-tooltip__value">
+              <div class="tooltip__value_label">
+                <div class="truncate">Loading...</div>
+              </div>
+              <div class="tooltip__value_wrapper">
+                <div class="tooltip__value_amount text-primary">
+                  <div>Loading...</div>
+                </div>
+                <div class="tooltip__value_detail">
+                  <div>(Loading..)</div>
+                </div>
+              </div>
+            </div>
+            <div class="map-tooltip__value">
+              <div class="tooltip__value_label">
+                <div class="truncate">Loading...</div>
+              </div>
+              <div class="tooltip__value_wrapper">
+                <div class="tooltip__value_amount text-primary">
+                  <div>Loading...</div>
+                </div>
+                <div class="tooltip__value_detail">
+                  <div>(Loading...)</div>
+                </div>
+              </div>
+            </div>
+            <div class="map-tooltip__value">
+              <div class="tooltip__value_label">
+                <div class="truncate">Loading...</div>
+              </div>
+              <div class="tooltip__value_wrapper">
+                <div class="tooltip__value_amount text-primary">
+                  <div>Loading...</div>
+                </div>
+                <div class="tooltip__value_detail">
+                  <div>(Loading...)</div>
+                </div>
+              </div>
+            </div>
+            <div class="tooltip__notch"></div>
+          </div>
+        </div>
```

Comment:
I don't understand what this change does, was this supposed to be added or did it slip in accidentally?